test/tests/functional/pbs_highreslog.py

```diff
@@ -44,6 +44,34 @@ class TestHighResLogging(TestFunctional):
     tm_micro_re = re.compile(
         r'(\d{2,2}/\d{2,2}/\d{4,4}\s\d{2,2}:\d{2,2}:\d{2,2}.\d{6,6})')
 
+    def validate_trace_job_lines(self, jid=None):
+        lines = self.server.log_lines(logtype='tracejob', id=jid, n='ALL')
+        if len(lines) > 1:
+            # Remove first line as it is not log line
+            lines = lines[1:]
+        for line in lines:
+            m = self.tm_micro_re.match(line)
+            if m is None:
+                # If this is accounting log, ignore it as
+                # Accounting log does not have high res logging
+                line = line.split()
+                _msg = 'Not Found high resolution time stamp in log'
+                self.assertFalse(len(line) >= 3 and
+                                 (line[2].strip() != 'A'), _msg)
+        _msg = self.server.shortname + \
+            ': High resolution time stamp found in log'
+        self.logger.info(_msg)
```

Comment:
if len(lines) < 1 (line 49 above) or if it has only 1 line (so that lines[1:] is None), you will still print this log message, are you sure that's what you want the test to do?

Reply:
If there is only one line the test will fail at assertFalse() before it executes this line. Also if there is a job running, which we are checking, there will always be atleast one line in the tracejob output.